pytorch_to_returnn/converter/converter.py

```diff
@@ -159,6 +159,7 @@ def get_returnn_config_serialized(self) -> str:
     config = dim_tags_proxy.collect_dim_tags_and_transform_config(config)
 
     code_lines = [
+      "import numpy\n",
```

Comment:
Why is this needed?

Reply:
If we have a `ConstantLayer`, it potentially contains `numpy.array[...]`.